src/routes/About.tsx

```diff
@@ -8,60 +8,64 @@ import AwsImage from "../assets/images/aws.png";
 import ReactImage from "../assets/images/react.png";
 import FlaskImage from "../assets/images/flask.png";
 import SqlImage from "../assets/images/sql.png";
+import { Link } from "react-router-dom";
+import { NONAME } from "dns";
 
 interface MemberProps {
   name: string;
   role: string;
+  github: string;
```

Comment:
generic descriptor? github what? is this a username? email? **_link_**?

Reply:
its actually a github _**suppository**_